sdk/communication/Azure.Communication.Identity/CHANGELOG.md

```diff
@@ -1,8 +1,13 @@
 # Release History
 
-## 1.3.0-beta.1 (Unreleased)
+## 1.3.0 (Unreleased)
 
 ### Features Added
+- Introduction of new scopes for token generation.
+    - `chat.join` (Access to Chat APIs but without the authorization to create, delete or update chat threads)
```

Comment:
```suggestion
    - `ChatJoin` (Access to Chat APIs but without the authorization to create, delete or update chat threads)
```
let's mention the scopes as defined in CommunicationTokenScope, same comment for the readme file